packages/react-debug-tools/src/ReactDebugHooks.js

```diff
@@ -20,7 +20,7 @@ import type {
 } from 'react-reconciler/src/ReactInternalTypes';
 import type {OpaqueIDType} from 'react-reconciler/src/ReactFiberHostConfig';
 
-import type {SuspenseConfig} from 'react-reconciler/src/ReactFiberSuspenseConfig';
+import type {SuspenseConfig} from 'react-reconciler/src/ReactFiberTransition';
```

Comment:
Nit: this new file name seems more generic than before, is there something more specific?

`ReactFiberCurrentTransitionState`? `ReactFiberCurrentTransitionValue`? `ReactFiberTransitionUpdate`?

Reply:
Named it this way because I anticipate we're going to put more stuff in that same module